deploy/docker/seqr/config/gunicorn_config.py

```diff
@@ -3,6 +3,4 @@
 workers = 1
 loglevel = 'info'
 timeout = 3600   # seconds (default is 30)
-errorlog = '-'  # '${INSTALL_DIR}/logs/gunicorn-error.log'
-accesslog = '-' # '${INSTALL_DIR}/logs/gunicorn-access.log'
```

Comment:
this was causing us to get duplicate logs in stackdriver for every URL that was hit. Since we have the robust django access logs, we don't also want the less-complete duplicate gunicorn logs